docs/articles/documentation/guides/continuous-integration/github-actions.md

```diff
@@ -49,6 +51,44 @@ jobs:
 
 This job runs on a GitHub-hosted virtual machine with the latest Windows version. `test` is the [job ID](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) that must be unique to the `jobs` object.
 
+> You can use a GitHub-hosted virtual machine with a variety of operating systems to run tests, as listed on [this page](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on). For simplicity, all examples in this article run on `windows-latest`.
+
+Github Actions uses the `macOS Catalina 10.15` virtual environment with *"System Integrity Protection"* enabled as `macos-latest`. With this setting enabled, TestCafe requires a screen recording permission, which is unobtainable programmatically. For this reason, TestCafe is unable to run tests locally on `macos-latest` in GitHub Actions.
+
+However, tests can run on macOS VMs if you connect the browser as remote.
```

Comment:
Is "VMs" short for "virtual machines"? If so, I think you should spell it out.

Reply:
That's right, thanks

Reply:
Thanks, please apply the change.

Reply:
done